solutions/usaco-809.mdx

```diff
@@ -2,14 +2,99 @@
 id: usaco-809
 source: USACO Bronze 2018 Febuary
 title: Taming the Herd
-author: Sathvik Chundru
+author: Sathvik Chundru, Aditya Gupta
 ---
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_taming_bronze_feb18.html)
 
 **Time Complexity:** $\mathcal{O}(N)$
 
 <LanguageSection>
+	
+<CPPSection>
+
+```cpp
+#include <iostream>
+#include <bits/stdc++.h>
+
+using namespace std;
+
+int main() {
+	freopen("taming.in", "r", stdin);
+	freopen("taming.out", "w", stdout);
+	int n;
+	cin >> n;
+	/*
+	 * A status of 2 means a breakout happened on day n.
+	 * A status of 1 means a breakout did not happen on day n.
+	 * A status of 0 means we are not sure what happened on day n.
+	 */
+	vector<int> status(n, 0);
+	status[0] == 2;  // a breakout must occur on the first day
```

Comment:
this line does nothing?

Reply:
which line? line 33 is because the problem says day 1 has a breakout

Reply:
no like that line doesn't do anything
did you mean `==` instead of `=`?

Reply:
Just fixed that, I think I already did that somewhere else in the code and forgot to delete this line.